packages/calypso-products/src/main.ts

```diff
@@ -174,6 +174,7 @@ export function planHasAtLeastOneFeature( plan: string | Plan, features: string[
  */
 export function getAllFeaturesForPlan( plan: Plan | string ): TranslateResult[] {
 	const planObj = getPlan( plan );
+	// console.log( 'planObj: Products Included', planObj.getProductsIncluded() );
```

Comment:
I'm sure you plan to but so that we don't forget: we probably want to remove this and the other console.logs in this file.

Reply:
Always a good reminder!  The code's not ready for prime time yet, but hopefully I'll remember to scrub those when I'm done with them.